test/server/cards/02_SHD/units/OmegaPartOfTheSquad.spec.ts

```diff
@@ -0,0 +1,65 @@
+describe('Omega, Part of the Squad', function() {
+    integration(function(contextRef) {
+        describe('Omega\'s Ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['omega#part-of-the-squad'],
+                        deck: ['crosshair#following-orders', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine'],
+                        base: 'echo-base',
+                        leader: 'hera-syndulla#spectre-two'
+                    }
+                });
+            });
+
+            it('can draw a clone', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.omega);
+                expect(context.player1).toHavePrompt('Select a card to reveal');
+                expect(context.player1).toHaveEnabledPromptButton(context.crosshair.title);
+                expect(context.player1).toHaveDisabledPromptButtons([context.atst.title, context.battlefieldMarine.title, context.cartelSpacer.title, context.pykeSentinel.title]);
+                context.player1.clickPrompt(context.crosshair.title);
+                expect(context.crosshair).toBeInLocation('hand');
+                expect(context.getChatLogs(2)).toContain('player1 takes Crosshair');
+
+                // Omega should cost 4 since it cannot discount itself
+                expect(context.player1.countExhaustedResources()).toBe(4);
+
+                context.player2.passAction();
```

Comment:
Can we play a second clone this round just to be extra safe

Reply:
Sorry, I misread this as `moveToNextActionPhase()`. Could we then add that and have a second action phase to confirm the ability resets? That's what I thought you were doing already, my b

Reply:
Nvm just saw it has that, ignore me lol